src/InfRcTransport.cc

```diff
@@ -150,15 +150,16 @@ namespace {
  *
  * \param context
  *      Overall information about the RAMCloud server or client.
- * \param sl
+ * \param my_sl
```

Comment:
why the old school perl style `my` variable name here?

Reply:
I can use a different variable. I'm trying to indicate, per conversation w Andy, that the sl is for the "current machine" and not the machine we're making a transport towards. Also, sl is not provided for rc-client at moment.

I can change variable to indicate "current machine" and explain that when we make a Transport towards another machine, this is null.

Reply:
Will go with "local_sl" with the comment explaining the reasoning behind the name